features/networking/ovn.feature

```diff
@@ -0,0 +1,147 @@
+Feature: OVN related networking scenarios
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-29954
+  @admin
+  @destructive
+  Scenario: Creating a resource in Kube API should be synced to OVN NB db correctly even post NB db crash too
+    Given the env is using "OVNKubernetes" networkType
+    Given I have a project
+    And I obtain test data file "networking/list_for_pods.json"
+    When I run the :create client command with:
+      | f | list_for_pods.json |
+    Then the step should succeed
+    Given 2 pods become ready with labels:
+      | name=test-pods |
+    And evaluation of `pod(0).name` is stored in the :pod1_name clipboard
+    And evaluation of `pod(1).name` is stored in the :pod2_name clipboard
+    #Checking whether Kube API data is synced on OVN NB db which in this case are couple of pods
+    Given I store the ovnkube-master "north" leader pod in the clipboard
+    And evaluation of `pod.ip_url` is stored in the :ovn_nb_leader_ip clipboard
+    And evaluation of `pod.node_name` is stored in the :ovn_nb_leader_node clipboard
+    And admin executes on the pod:
+      | bash | -c | ovn-nbctl list logical_switch_port |
+    Then the step should succeed
+    And the output should contain:
+      | <%= cb.pod1_name %> |
+      | <%= cb.pod2_name %> |
+    #Simulating a NB db crash
+    Given I use the "<%= cb.ovn_nb_leader_node %>" node
+    And I run commands on the host:
+      | pkill -f OVN_Northbound |
```

Comment:
normally `SIGKILL` or `SIGSEGV` would be more appropriate, because `SIGTERM` is usually handled.

It looks like the only signal handler in the OVS source is for `SIGSEV` which prints out a backtrace, but I'm not sure.

https://github.com/openvswitch/ovs/commit/e2ed6fbeb18cba1235c6a260b754f11ecc9d4cb0#diff-37c472540fb42eff036bcb7639176986R212

probably safer to do a `pkill --SIGKILL` to be sure OVS isn't handling anything.

or is the test already working okay?

Reply:
@rbbratta Yep, I have been testing it with pkill with no issues. So seems good to me for now.